.github/workflows/ci.yaml

```
@@ -74,6 +74,6 @@ jobs:
           go-version-file: './go.mod'
           check-latest: true
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@639cd343e1d3b897ff35927a75193d57cfcba299 # v3.6.0
+        uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          version: v1.53
+          version: v1.54.1
```

thanks for this PR

please set to 
```suggestion
          version: v1.54
```

with that we dont need to bump for patch releases

---

Done.